openwisp_monitoring/device/tests/test_models.py

```diff
@@ -578,6 +578,90 @@ def _create_env(self):
         )
         return dm, ping, load, process_count
 
+    def test_disabling_critical_check(self):
+
+        Check = load_model('check', 'Check')
+        dm, ping, load, process_count = self._create_env()
+
+        ping_check_instance = Check.objects.create(
+            name='Ping Check',
+            check_type='ping',
+            content_object=dm.device,
+            params={},
+        )
+        dm.update_status('ok')
+
+        self.assertEqual(dm.status, 'ok')
+        with catch_signal(health_status_changed) as handler:
+            ping_check_instance.is_active = False
+            ping_check_instance.save()
+        if handler.call_count > 0:
```

Comment:
why this if statement? Please replace this with just asserting that it has been called once, then proceed with the other assertions below which look good.